src/scp/test/SCPTests.cpp

```diff
@@ -308,6 +308,13 @@ class TestSCP : public SCPDriver
     {
         return mSCP.getSlot(slotIndex, false)->getNominationLeaders();
     }
+
+    std::chrono::milliseconds
+    computeTimeout(uint32 roundNumber, bool isNomination) override
```

Comment:
Do we need to test with a few different timeout values for sanity checking? (we've never updated these before, so there might be bugs hiding that rely on the default values)

Reply:
thanks! The matrix might be too aggressive though, as it causes a combinatorial explosion of SECTIONs (it shows up in our CI too, the build for this PR takes ~1.5 hours). We can reduce this to 3 combinations: current network config, minimum network config and maximum network config.